Sources/SwiftInspectorVisitors/Tests/PropertySyntaxVisitorSpec.swift

```diff
@@ -0,0 +1,338 @@
+// Created by francisco_diaz on 7/7/21.
+//
+// Distributed under the MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import Foundation
+import SwiftInspectorTestHelpers
+import Quick
+import Nimble
+
+@testable import SwiftInspectorVisitors
+
+final class PropertySyntaxVisitorSpec: QuickSpec {
```

Comment:
This is a copy/paste from https://github.com/fdiaz/SwiftInspector/pull/96/commits/0050b8d6add3d106a627d104742f21ad9ccf4379#diff-982ee3afe15209354e026059da96ee6332d9c4973e7b431ff9c37bc741d25e25L334